<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubeWidget</class>
 <widget class="QWidget" name="CubeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CubeWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>С</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="dateEditBeg">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>По</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="dateEditEnd">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="cmdUpd">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Обновить данные</string>
         </property>
         <property name="text">
          <string>Обновить</string>
         </property>
         <property name="icon">
          <iconset theme="view-refresh">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxY">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Ось Y</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxX">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Ось X</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonSum">
       <property name="text">
        <string>Сумма</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonAvg">
       <property name="text">
        <string>Среднее</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonMax">
       <property name="text">
        <string>Максимум</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonMin">
       <property name="text">
        <string>Минимум</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>olap/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
